env/staging/ecs/terragrunt.hcl

```diff
@@ -69,6 +69,7 @@ inputs = {
   memory                              = 1024
   aggregate_metrics_table_arn         = dependency.dynamodb.outputs.aggregate_metrics_arn
   schedule_expression                 = "rate(24 hours)"
+  metrics_token                       = "${get_env("TF_VAR_METRICS_TOKEN")}"
```

Comment:
I had thought the `TF_VAR_METRICS_TOKEN` environment variable would get automatically read in here, but I haven't tested with a terragrunt.hcl file.

Reply:
i'll test it out and see what happens when i remove that line and introduce a TF_VAR_metrics_token. Maybe the case is the reason why it wasn't working and forced me to do this.